releng/platform-definitions/platform-definition-2020-12/platform-definition-2020-12.target

```diff
@@ -0,0 +1,68 @@
+   provided that the following conditions are met:
+
+   1. Redistributions of source code must retain the above copyright notice, this list of conditions
+   and the following disclaimer.
+
+   2. Redistributions in binary form must reproduce the above copyright notice, this list of
+   conditions and the following disclaimer in the documentation and/or other materials provided with
+   the distribution.
+
+   3. Neither the name of the copyright holder nor the names of its contributors may be used to
+   endorse or promote products derived from this software without specific prior written permission.
+
+   THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR
+   IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
+   FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR
+   CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+   DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+   DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+   WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY
+   WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+<?pde version="3.8"?>
+<target name="jmc-target-2020-12" sequenceNumber="47">
+    <locations>
+        <location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
+            <unit id="com.sun.mail.jakarta.mail" version="1.6.5"/>
+            <unit id="com.sun.activation.jakarta.activation" version="1.2.1"/>
+            <unit id="org.owasp.encoder" version="1.2.2"/>
+            <unit id="org.lz4.lz4-java" version="1.7.1"/>
+            <unit id="org.twitter4j.core" version="4.0.7"/>
+            <unit id="org.hdrhistogram.HdrHistogram" version="2.1.12"/>
+            <unit id="org.adoptopenjdk.jemmy-awt-input" version="2.0.0"/>
+            <unit id="org.adoptopenjdk.jemmy-browser" version="2.0.0"/>
+            <unit id="org.adoptopenjdk.jemmy-core" version="2.0.0"/>
+            <unit id="org.adoptopenjdk.jemmy-swt" version="2.0.0"/>
+            <repository location="http://localhost:8080/site"/>
+        </location>
+        <location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
+            <unit id="org.eclipse.equinox.executable.feature.group" version="3.8.1000.v20200915-1508"/>
+            <unit id="org.eclipse.pde.feature.group" version="3.14.600.v20201202-1800"/>
+            <unit id="org.eclipse.platform.sdk" version="4.18.0.I20201202-1800"/>
+            <repository location="http://download.eclipse.org/releases/2020-12/"/>
```

Comment:
Try using https:// to workaround the http -> https redirect issue at download.eclipse.org.

Reply:
Sorry I had hoped the problem was only http to https redirects. We'll need webmaster@eclipse.org to fix it. My guess is the fix for https://bugs.eclipse.org/bugs/show_bug.cgi?id=569730 broke it, so you can follow along there. 

Sorry for the inconvenience.

Reply:
Can you retrigger your build? IIUC the badly behaved redirect has been resolved.  https://bugs.eclipse.org/bugs/show_bug.cgi?id=569765

Reply:
Thanks! It works now, but I think we might be hitting https://bugs.eclipse.org/bugs/show_bug.cgi?id=569039.